src/program.js

```diff
@@ -34,7 +34,7 @@ export class Program {
       if (!commandOptions) {
         return;
       }
-      return yargs.options(commandOptions);
+      return yargs.env('WEB_EXT').options(commandOptions);
```

Comment:
Also, I think you should add a comment saying that this might be fixed in the future by https://github.com/yargs/yargs/issues/486